src/index.ts

```diff
@@ -48,5 +48,5 @@ export * from "./modules/phoneNumber";
 // TimeAgo
 export { default as timeAgo } from "./modules/timeAgo";
 
-// RemainedTime
-export { default as RemainedTime } from "./modules/remainedTime";
+// RemainingTime
+export { default as RemainingTime } from "./modules/remainingTime";
```

Comment:
please convert every single variable of your code to the `camelCase` convention.

Reply:
Converted all the variables.